build/build.kern.mk

```diff
@@ -25,11 +25,12 @@ include $(TOPDIR)/config.mk
 SOURCES_ALL = $(SOURCES)
 SOURCES_ALL += $(foreach var, $(CONFIG_OPTS), $(value SOURCES-$(var)))
 
+include $(TOPDIR)/build/flags.kern.mk
```

Comment:
That's because in `build/flags.kern.mk` the `KFTRACE` and `KGPROF` flags are defined. That makes it possible to include the files saved under `SOURCES_KFTRACE`  and `SOURCES_KGPROF` to `SOURCES` variable.

Reply:
Got it. Thanks!